content/concept-transaction-cost.md

```diff
@@ -7,24 +7,74 @@ Every transaction must [specify how much XRP it will destroy](#specifying-the-tr
 
 ## Current Transaction Cost ##
 
-The current transaction cost required by the network is typically **0.01 XRP** (10,000 drops), although it sometimes increases due to network load.
+The current transaction cost required by the network for a standard transaction is typically **0.01 XRP** (10,000 drops), although it sometimes increases due to network load.
 
 You can also [query `rippled` for the current transaction cost](#querying-the-transaction-cost).
 
+### Special Transaction Costs ###
+
+Some transactions have different transaction costs:
+
+| Transaction           | Cost Before Load Scaling |
+|-----------------------|--------------------------|
+| Reference Transaction (Standard cost of most transactions) | 10 drops |
```

Comment:
Rather than revise this section (which already has "before load scaling" in the table header) I revised the section about the Current Transaction Cost to clarify that the 0.01 XRP number is after load scaling.